test/test_utils.py

```diff
@@ -359,6 +360,18 @@ def test_write_stan_json_bad(self):
 
 
 class ReadStanCsvTest(unittest.TestCase):
+    # pylint: disable=invalid-name
+    @contextlib.contextmanager
+    def assertRaisesRegexNested(self, exc, msg):
```

Comment:
great - many thanks!

Reply:
This is a little brittle because the traceback also contains the lines that triggered the error, which _in theory_ could match the regex. I think it's nearly impossible for this to happen in a well specified test though.

Reply:
I've resolved the above isssue